app/views/exports/hamilton_pooling_plate_pbmc.csv.erb

```diff
@@ -31,10 +31,14 @@
 %>
 <%
   # Calculate the resuspension volumes from the pooled samples count
+  resuspension_volume_per_sample = 2.2
```

Comment:
we tend to put numbers like these in the purpose config rather than hardcode them like this, easier to find if you want to change later

Reply:
on the other hand it's a pretty specific csv file